src/Modal/Modal.jsx

```diff
@@ -68,7 +70,7 @@ class Modal extends Component {
           <Overlay onClick={onClose} />
           <Height>
             <Row position="center" verticalAlign="middle">
-              <Col>
+              <Col lg={modalLength}>
```

Comment:
this should be the width property then right? 

I wonder if it's smarter to just have 2 different size of modals and let the consumer choose either large or small. 
Small would have the buttons centered and be this wide, large would be what we currently have. 

The advantage of this way would be that we retain full control over how the modal would be shown in different screen widths (ie there should be no difference between large/small for xs screen sizes). If we let the users pass these values in then they will have trouble when they want to change modal size based on screen size, and will need to pass even more props.

Reply:
makes sense but if design later on has another length how will we proceed?

Reply:
we can discuss at that point if it makes sense to have a third width, and even then we would have less than the 7 we would have if we allow the user to pass it in. Same reasoning why we don't allow various button sizes or colours to be configured. We also need to be aware of the creep of multiple components and try to ensure that designs use the existing components where possible, or commit to a global change - otherwise we will have a very fragmented UI library and a very fragmented user experience across our apps